Update endorsement procedure

stotzjohn opened the pull request:
This will update the endorsement procedure to make it more public and allow for better discussion on the candidates.

stotzjohn committed:
Allow others into the endorsement meeting

This would allow members of The Poly that aren't as involved and are in coordinator positions to still be able to participate. This is useful as they may have valuable input and would allow for a better interview.

stotzjohn committed:
Public Endorsement Meetings

Updated to allow for endorsement meetings to be made public after the meeting is complete. The meeting is still closed to prevent the general RPI campus from participating as they have the debates to ask the candidates questions.

stotzjohn committed:
Made Endorsements Public 

Fixed some spelling

stotzjohn committed:
Move Endorsement from Constitution to Bylaws

The constitution should outline what The Poly has the right to do while the Bylaws should have procedures on how to do it.

stotzjohn committed:
Add endorsement procedure

The bylaws are the correct location as the bylaws should outline the procedure of how The Poly will go about following the rules granted in the constitution.

stotzjohn committed:
Add endorsement procedure

bparslow commented on Bylaws.md:
```diff
@@ -104,3 +104,42 @@ The purpose of _The Polytechnic_ shall be to keep the members of the Rensselaer
+        
+        1.	Meetings during which candidates are interviewed shall be closed to all except those outlined in Section (IV) subsection (3) clause (iii) and the interviewee. Quorum shall consist of two-thirds of the editorial board membership. 
+        
+        1.  The meeting which candidates are interviewed shall not be confidential. 
```
"The meetings in which candidates are interviewed shall not be confidential." j grammar stuff

bparslow commented on Bylaws.md:
```diff
@@ -104,3 +104,42 @@ The purpose of _The Polytechnic_ shall be to keep the members of the Rensselaer
+        
+        1.	The editorial board shall provide the option of an interview to each candidate as recognized by the Student Senate running for an office up for endorsement and present to them prior to the interview a written questionnaire. The editorial board must consider each of those candidates who are interviewed for endorsement.
+        
+        1.	Meetings during which candidates are interviewed shall be closed to all except those outlined in Section (IV) subsection (3) clause (iii) and the interviewee. Quorum shall consist of two-thirds of the editorial board membership. 
```
Why not just add here instead of the confidentiality thing "The editorial board may publish transcripts or recordings of these interviews as part of its coverage of elections."

bparslow commented on Bylaws.md:
```diff
@@ -104,3 +104,42 @@ The purpose of _The Polytechnic_ shall be to keep the members of the Rensselaer
+
+            1. Neither absentee ballots nor voting by proxy are allowed for referenda endorsement meetings.
+            
+    1. All editoral board members may participate in endorsement meetings. Other memebers of _The Poly_ may be invited to participate by the disgrssion of the editor in chief. The editorial board may reject the invite by a 2/3 vote.
```
"disgrssion" and also i totally disagree with rejecting the invite by a 2/3 vote

stotzjohn committed:
Edit endorsement procedure

Made it so the invitations to the editorial board must be announced two business meetings before the first endorsement meetings. 

Also corrected some grammatical errors.

bparslow committed:
Fix spelling and wording

bparslow committed:
Move sentence to more relevant spot

stotzjohn commented:
Motion passes in 2-17-2020 business meeting 9-0-0
